habinezadalvan/Epic-Mail
# #164582524 fixes malfunctions on the UI

**what does this PR do?**
Fixes malfunctions in that were found in the UI.
**Description of Task to be completed**
-Make login text turns white when the user hovers on buttons.
-Remove margins and paddings to menu buttons to almost every page
-remove margins and padding on emails to all pages with emails.

**How should this be tested manually**
-When a user browsers to epic mail welcome page, he can hover over the login button and immediately the text turns white. 
- after login he can see that there are no longer margins and padding on menu buttons and messages on hover.
**relative pivotal tracker story**
#164582524
